Assets/Scripts/StateManager.cs

```diff
@@ -159,13 +159,25 @@ List<AttackLog> ApplyAndRecordDamage(Piece piece, ICollection<MoveData> attacker
         List<AttackLog> attackLogs = new List<AttackLog>();
         foreach (MoveData attackerMoveData in attackerMoveDatas)
         {
-            int damage = baseDamage;
+            double damage = baseDamage;
             // apply enemy specific multipliers here
+            // damage dampening if outnumbered
+            switch (attackerMoveDatas.Count)
+            {
+                case 2:   // 40% on 2 enemies
+                    damage *= 0.4;  break;
+                case 3:   // 20% on 3 enemies
+                    damage *= 0.2;  break;
+                case 4:   // 5% on 4 enemies
+                    damage *= 0.05; break;
+            }
+            if (piece.IsCounteredBy(attackerMoveData.piece)) damage *= 1.2;
+            damage *= piece.health / 100.0;

```

Comment:
This assumes max health is 100. Could we have an additional field in `Piece`?

Reply:
Sure, added Piece.maxHealth in recent commit.